flowapi/flowapi/main.py

```diff
@@ -42,7 +42,12 @@
 async def connect_logger():
     log_level = current_app.config["LOG_LEVEL"]
     log_root = current_app.config["LOG_DIRECTORY"]
-    current_app.logger.setLevel(logging.getLevelName(log_level))
+    logger = logging.getLogger("flowapi")
+    logger.setLevel(log_level)
+    ch = logging.StreamHandler()
+    ch.setLevel(log_level)
+    logger.addHandler(ch)
+    current_app.log = structlog.wrap_logger(logger)
```

Comment:
This may just be personal preference, but I find `current_app.logger` slightly more readable than `current_app.log` (perhaps because it doesn't have the noun vs verb ambiguity). Any objections to changing it back to `current_app.logger`?

Reply:
Can't, quart has special rights over `app.logger`. Could change to `debug_logger`?

Reply:
Ah, I see. Perhaps `flowapi_logger` then (since we may not just be logging debug messages)?

As an aside, this seems like a possibility for a trip-up. I was unaware that quart uses `app.logger` internally. Probably good to add a comment to that effect in `flowapi/main.py` where we set up the logger?